### DISCUSSION TITLE
How to provide authentication

### DISCUSSION THREAD

**Discussion by fnuky** [HaishinKit/HaishinKit.swift#1243] [Q&A]
Hi, I went trough whole SDK how to provide authentication for RTMP streamer. I have RTMP server and stream name with username and password. Tries to connect to RTMP everytime fails.

description	String	"[ AccessManager.Reject ] : [ code=403 need auth; authmod=llnw ] : "	

I tryied also put username and password into RTMP url but it doesn't work.. 

Can anyone help me please?

**Comment by shogo4405** [comment]
Unfortunately, HaishinKit does not support llnw format authentication.
